src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -34,7 +37,28 @@ public function createArgumentMetadata($controller): array
         }
 
         foreach ($reflection->getParameters() as $param) {
-            $arguments[] = new ArgumentMetadata($param->getName(), $this->getType($param, $reflection), $param->isVariadic(), $param->isDefaultValueAvailable(), $param->isDefaultValueAvailable() ? $param->getDefaultValue() : null, $param->allowsNull());
+            $attribute = null;
+            if (method_exists($param, 'getAttributes')) {
```

Comment:
We've use `\PHP_VERSION_ID >= 80000` in other attribute-related code instead.

Reply:
indeed, fixed in f1f37a899c